examples/linearCylinderPlaneStrain/linearCylinderPlaneStrain.m

```diff
@@ -0,0 +1,149 @@
+%md# Linear cylinder plane strain  example  
+%md
+%md[![Octave script](https://img.shields.io/badge/script-url-blue)](https://github.com/ONSAS/ONSAS.m/blob/master/examples/linearPlaneStrain/onsasExample_linearPlaneStrain.m)
+%md
+%md In this example a hollow cylinder submitted to an internal pressure $p_i$ Pa as shown in the diagram depcited below is considered. The length of the cylinder is $L_z $ m and the internal and external radios are $R_i$ m and $R_e$ m, respectively. 
+%md
+%md```@raw html
+%md<img src="../../assets/linearCylinderPlaneStrain/ilusCylinderPlaneStrain.svg" alt="linear cylinder diagram" width="500"/>
+%md```
+%md
+%md A cylindrical system of coordinates is defined considering the unitary vectors ($e_r$, $e_\theta$, $e_z$). 
+%md The material employed is isotropic and homogeneous with elasticity modulus $E=1$ MPa and Poisson's coefficient $\nu=0.3$. The pressure is radial and applied on the internal surface $\mathbf{\mathit{p_i}} =  - p_i \, e_r$. The boundary conditions correspond to this plane strain example holds:
```

Comment:
Poisson's ratio https://en.wikipedia.org/wiki/Poisson%27s_ratio

Reply:
"The boundary conditions corresponding to this plane strain example are:"

Reply:
i think that there is a sign error in the internal pressure... you might be thinking about the normal vector... but the tension load vector is towards er...

Reply:
Done